### PR TITLE
Event and newsitem blocks views

### PR DESCRIPTION
**Pull request by nzambello**
Rel: https://github.com/plone/plone.volto/pull/35 https://github.com/plone/plone.volto/issues/32

With blocks behavior enabled by default on Event and News Item CTs, now we handle blocks rendering in theirs views.

Separated Event-specific metadata fields into separate component to reuse in both "classic" metadata view and the blocks view

### CODE REVIEW

File: src/components/theme/View/EventView.jsx [hunk 1]
```diff
@@ -72,99 +23,43 @@ const EventView = ({ intl, content }) => (
...
+  const { content } = props;
+
+  return (
+    <div id="page-document" className="ui container viewwrapper event-view">
+      <Grid>
+        <Grid.Column mobile={12} tablet={7} computer={7}>
+          {hasBlocksData(content) ? (
+            <RenderBlocks {...props} />
+          ) : (
+            <EventMetadataView {...props} />
```
Comment (tiberiuichim) [inline src/components/theme/View/EventView.jsx:52]: I think the EventMetadataView should always be shown together with the blocks.

**Review by sneridagh — APPROVED** [review]
We should merge plone.volto PRs first.

File: src/components/theme/EventDetails/EventDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
+import React from 'react';
```
Comment (nileshgulia1) [inline src/components/theme/EventDetails/EventDetails.jsx:1]: Is `EventDetails.jsx` is duplicated? We have two of them here.
-cc @nzambello

**Comment by sneridagh** [comment]
@nzambello What's the status of this one?

**Comment by nzambello** [comment]
> @nzambello What's the status of this one?

we should address previous comments, but since the plone.volto pr is merged, once completed this is ready.

open tasks:
- naming on EventDetails
- decide when/how to show the new component

**Comment by sneridagh** [comment]
@plone/volto-team we need to merge this, any objection? Can someone work on the remaining issues/tasks?

I overlooked that the last plone.volto already enables the blocks in there for Events/News, but we don't have the Volto counterpart.

**Comment by giuliaghisini** [comment]
for me it's ok

> @plone/volto-team we need to merge this, any objection? Can someone work on the remaining issues/tasks?
> 
> I overlooked that the last plone.volto already enables the blocks in there for Events/News, but we don't have the Volto counterpart.

**Review by tiberiuichim — APPROVED** [review]
LGTM

**Comment by ksuess** [comment]
The event meta data is important for an event. On small screens I would prefer to see these right after the title, rather than at the end.

localhost_3000_volto-test-news(iPhone 12 Pro) (1)

**Pull request merged**